src/inspect/mod.rs

```diff
@@ -65,7 +67,19 @@ impl TypeHint {
     /// ```
     pub const fn module_attr(module: &'static str, attr: &'static str) -> Self {
         Self {
-            inner: TypeHintExpr::ModuleAttribute { module, attr },
+            inner: if str_equals(module, "builtins") {
```

Comment:
for what it's worth I discovered this alternative the other day:


```suggestion
            inner: if matches!(module.as_bytes(), b"builtins") {
```

I don't know if that's any nicer than just defining the helper function, wdyt?

Reply:
Looks nicer. Nice tip! Done.